GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,156 @@
+        * Click [+/Add] and enter values for the following fields:
+            * Subscription: Your preferred enterprise subscription for Cobalt
+            * Resource group: Create new (ex. cobalt-devint-admin-rg)
+            * Storage account name: (ex. cobalttfstates)
+        * Click [Review+Create] then [Create]
+        * Once deployment for storage account is completed, go to the resource and visit the Blobs sub-menu
+        * Click [+Container] then create a container name (ex. az-hw-remote-state-container ) with private access
+
+3. Configure Azure Devops pipeline using Azure resource values from previous steps
+
+    * Add the Azure Subscription being used for Cobalt as a *Service Connection*
+        * Return to your Azure DevOps subscription
+        * Find and select the Project Settings tab at the bottom of the screen
+        * Under the Pipelines menu select Service Connections
+        * From the Service Connections menu, select [+New Service Connection]
+        * Choose Azure Resource Manager from the dropdown and enter the Azure Subscription being used for Cobalt
+
+    * Configure *Infrastructure Pipeline Variables*
+        * Select Pipelines tab from within side-navigation menu then select Library tab
+        * Click [+Variable group] and add the following variables:
+
+            | Name   | Value | Description
+            |-------------|-----------|-----------|
+            | `AGENT_POOL` | Hosted Ubuntu 1604 | The type of build agent used for your deployment. |
+            | `ARM_PROVIDER_STRICT` | false | Terraform ARM provider modification |
+            | `BUILD_ARTIFACT_NAME` | drop | Name to identity the folder containing artifacts output by a build. |
+            | `GO_VERSION`| 1.12.5 | The version of Go terraform deployments are bound to. |
+            | `PIPELINE_ROOT_DIR` | 'devops/providers/azure-devops/templates/' | A path for finding Cobalt templates. |
+            | `REMOTE_STATE_CONTAINER` | az-hw-remote-state-container | The remote storage container name for managing the state of a cobalt template's deployed infrastructure. Also is used as a naming convention for branching state into multiple workspaces. |
+            | `SCRIPTS_DIR` | infrastructure/scripts | Path to scripts used at runtime for composing build and release jobs at various pipeline stages. |
+            | `TEST_HARNESS_DIR` | test-harness/ | A path to the cobalt test harness for running integration and unit tests written in Docker and Golang. |
+            | `TF_ROOT_DIR`| infra | The primary path for all Cobalt templates and the module they are made of.  |
+            | `TF_VERSION`| 0.12.4 | The version of terraform deployments are bound to. |
+            | `TF_WARN_OUTPUT_ERRORS`| 1 | The severity level of errors to report |
+
+    > Important: Every targeted environment specified within the build pipeline expects a
+    > variable group specified with the naming convention `<ENVIRONMENT_NAME> Pipeline Variables`
```

Comment:
This is incorrect. Should be *<ENVIRONMENT_NAME> Environment Variables* (also references below)

Reply:
updated